release-notes/swan-lake-2201.1.0-release-note.md

```diff
@@ -30,9 +30,364 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### New Features
 
+##### Support for the spread operator in the list constructor
+
+Introduced the spread operator support for the list constructor expression.
+
+If the spread operator in a list constructor expression is `...x`, then, `x` is expected to be a list (i.e., an array or a tuple). All the member values of the list that result from evaluating `x` are included in the list value being constructed.
```

Comment:
```suggestion
If the `spread` operator in a `list` constructor expression is `...x`, then, `x` is expected to be a list (i.e., an array or a tuple). All the member values of the list that result from evaluating `x` are included in the list value being constructed.
```

Reply:
Spread and list are not ballerina specific keywords

Reply:
Better to use backticks as it might be confusing with the verb form of it.